src/exe/cimbar_recv2/recv2.cpp

```diff
@@ -39,14 +39,10 @@ int main(int argc, char** argv)
 {
 	cxxopts::Options options("cimbar video decoder", "Use the camera to decode data!");
 
-	unsigned colorBits = cimbar::Config::color_bits();
-	unsigned ecc = cimbar::Config::ecc_bytes();
 	unsigned defaultFps = 15;
 	options.add_options()
 		("i,in", "Video source.", cxxopts::value<string>())
 		("o,out", "Output directory (decoding).", cxxopts::value<string>())
-		("c,colorbits", "Color bits. [0-3]", cxxopts::value<int>()->default_value(turbo::str::str(colorBits)))
```

Comment:
Since cimbar_recv and recv2 do the same thing, I'm leaning towards having recv2 be the less configurable version. (e.g. less overrides for debugging, etc)

So more ad-hoc stuff like changing ecc (or colors, or whatever else) will only be supported in one, with this executable being a way to test the decoder C api functionality that the webapp does ... without the webapp.